File: MtgJsonToMssql/src/main/kotlin/organize/printings/wide/Identifiers.kt
package organize.printings.wide

import kotlinx.serialization.json.JsonElement
import organize.printings.stringOpt

data class Identifiers(private val jsonElement: JsonElement){
    val cardKingdomFoilId : String = jsonElement.stringOpt("cardKingdomFoilId")
    val cardKingdomId : String = jsonElement.stringOpt("cardKingdomId")
    val mcmId : String = jsonElement.stringOpt("mcmId")
    val mcmMetaId : String = jsonElement.stringOpt("mcmMetaId")
    val mtgArenaId : String = jsonElement.stringOpt("mtgArenaId")
    val mtgoFoilId : String = jsonElement.stringOpt("mtgoFoilId")
    val mtgoId : String = jsonElement.stringOpt("mtgoId")
    val mtgjsonV4Id : String = jsonElement.stringOpt("mtgjsonV4Id")
    val multiverseId : String = jsonElement.stringOpt("multiverseId")
    val scryfallId : String = jsonElement.stringOpt("scryfallId")
    val scryfallOracleId : String = jsonElement.stringOpt("scryfallOracleId")
    val scryfallIllustrationId : String = jsonElement.stringOpt("scryfallIllustrationId")
    val tcgplayerProductId : String = jsonElement.stringOpt("tcgplayerProductId")
}

File: MtgDiscoveryAzFuncs/MtgDiscoveryAzFuncs/dtos/Identifiers.cs
using Newtonsoft.Json;

namespace MtgDiscoveryAzFuncs.dtos
{
    public class Identifiers
    {
        [JsonConstructor]
        public Identifiers(
            [JsonProperty("scryfallId")] string scryfallId,
            [JsonProperty("multiverseId")] string multiverseId,
            [JsonProperty("cardKingdomId")] string cardKingdomId,
            [JsonProperty("cardKingdomFoilId")] string cardKingdomFoilId
        )
        {
            this.ScryfallId = scryfallId;
            this.MultiverseId = multiverseId;
            this.CardKingdomId = cardKingdomId;
            this.CardKingdomFoilId = cardKingdomFoilId;
        }

        [JsonProperty("scryfallId")]
        public string ScryfallId { get; }

        [JsonProperty("multiverseId")]
        public string MultiverseId { get; }

        [JsonProperty("cardKingdomId")]
        public string CardKingdomId { get; }

        [JsonProperty("cardKingdomFoilId")]
        public string CardKingdomFoilId { get; }
    }
}